### ISSUE TITLE
Pull firmware from MobiFlight-FirmwareSource github repo

### ISSUE THREAD

**Issue by neilenns** [MobiFlight/MobiFlight-Connector#525]
Add a script for pulling firmware builds from GitHub releases of the MobiFlight-FirmwareSource project.

**Comment by neilenns** [comment]
Ok @DocMoebiuz I messed around with this for a while and what I came up with is this.

When the firmware needs a new release you create one onthe firmware's github repo release page (https://github.com/MobiFlight/MobiFlight-FirmwareSource/releases) and tag it with a version (e.g. `v1.12.0`). GitHub will automatically build the firmware, take care of versioning it, and attach the binaries to the release page.

On the MobiFlight-Connector side all that changes is the script to run. Instead of running `build-firmware.bat` I have a new `get_firmware.ps1` script that grabs the latest firmware release from GitHub and drop it in the `firmware` folder. If necessary a specific version can be specified on the command line.

That's all that changes. You still modify `MobiFlightModuleInfo.cs` to update the latest firmware version, and the `Release` build in Visual Studio is still the same.

I haven't opened a PR for this yet but if you want to try the script yourself it's here: https://github.com/neilenns/MobiFlight-Connector/blob/neilenns/issue525/Build/get_firmware.ps1. Just put it in your `Build/` folder and run it from a PowerShell prompt.
